src/runtime/link-accounts-flow.js

```diff
@@ -194,41 +194,66 @@ export class LinkSaveFlow {
     /** @private @const {!Window} */
     this.win_ = deps.win();
 
+    /** @private @const {!./deps.DepsDef} */
+    this.deps_ = deps;
+
     /** @private @const {!web-activities/activity-ports.ActivityPorts} */
     this.activityPorts_ = deps.activities();
 
     /** @private @const {!../components/dialog-manager.DialogManager} */
     this.dialogManager_ = deps.dialogManager();
 
+    /** {!../api/subscriptions.SaveSubscriptionRequest} */
```

Comment:
`@type`. But why do we need a default request? Just for testing? If so, add `TODO` so we don't forget to remove.

Reply:
Done